lib/results_screen.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:distal_radius/image_handler.dart';
+import 'package:flutter/material.dart';
+
+import "screen_button.dart";
+
+import 'dart:io';
+
+class ResultsScreen extends StatefulWidget {
+  const ResultsScreen({super.key});
+
+  static const String id = "results_screen";
+
+  @override
+  State<ResultsScreen> createState() => _ResultsScreenState();
+}
+
+// State class that handles logic in the screen
+// contains all functions relevant to business logic
+class _ResultsScreenState extends State<ResultsScreen> {
+  ImageHandler imageHandler = ImageHandler();
+
+  void cancelResults() {
+    print("Cancelled");
+  }
+
+  void toExport() {
+    print("Send to export");
+  }
+
+  Widget getHeader() {
+    return Stack(
+      alignment: Alignment.center,
+      children: const [
+        Positioned(left: 10, child: BackButton()),
+        Align(
+            child: Text(
+          "Results",
+          textAlign: TextAlign.center,
+          textScaleFactor: 1.5,
+        )),
+      ],
+    );
+  }
+
+  Widget getImages() {
+    var frontImage = FileImage(File(imageHandler.frontImagePath!));
+    var sideImage = FileImage(File(imageHandler.sideImagePath!));
+
+    return Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+      Image(height: 300, width: 200, image: frontImage),
+      Image(
+        height: 300,
+        width: 200,
+        image: sideImage,
+      ),
+    ]);
+  }
```

Comment:
Again, the same issue as in the initial UI screen. The images overflow off the side of the screen.

Additionally, the larger images look much better, IMO. You might want to update the other screens to have these larger images.